lib/promoted/ruby/client/defaults.rb

```diff
@@ -0,0 +1,17 @@
+module Promoted
+  module Ruby
+    module Client
+      USE_CASES = {'UNKNOWN_USE_CASE'=> 0,
```

Comment:
Think should work okay.  Numbers are more efficient though.

Reply:
Eventually, we'll want to support protobuf serialization (which will handle minimizing this).